frontend/src/metabase/collections/containers/FormCollectionAndDashboardPicker/FormCollectionAndDashboardPicker.tsx

```diff
@@ -35,6 +36,11 @@ function ItemName({
   dashboardId: DashboardId;
   type?: "collections" | "snippet-collections";
 }) {
+  const { data: collection } = useGetCollectionQuery(
```

Comment:
This won't fire on root collection, but maybe that's alright, since it will still get the standard `collection` icon